src/python/pants/rules/core/test.py

```diff
@@ -200,14 +198,14 @@ class Test(Goal):
 
         coverage_reports = await MultiGet(
             Get[CoverageReport](
-                CoverageDataBatch, coverage_batch_cls(tuple(addresses_and_test_results))
+                CoverageDataBatch, coverage_batch_cls(tuple(addresses_and_test_results))  #type: ignore
```

Comment:
Nit: `# type: ignore`. Also preferred if you can find the specific error, e.g. `# type: ignore[call-args]`. (Remove this ignore, run `build-support/bin/mypy.py`, then look for the code.)